goose3/configuration.py

```diff
@@ -94,7 +94,7 @@ def __repr__(self):
 
 class AuthorPattern(object):
 
-    def __init__(self, *, attr=None, value=None, content=None, tag=None, subpattern=None):
+    def __init__(self, attr=None, value=None, content=None, tag=None, subpattern=None):
```

Comment:
I'm actually a C programmer, so I'm not exactly sure what the `*,` is :smile:. @timoilya is this the correct change to fix python 2.7 support?

Reply:
Yeah, for some reason I forgot about that when I was editing the `.travis.yml`

Reply:
Even though the `3` is in the name :smile: